integrations/vite/css-modules.test.ts

```diff
@@ -0,0 +1,68 @@
+import { describe, expect } from 'vitest'
+import { css, html, test, ts, txt } from '../utils'
+;['postcss', 'lightningcss'].forEach((transformer) => {
```

Comment:
I know we do this in other places, but I think a normal for loop is a bit cleaner here (gets rid of the `;` and allows a new line again)

```suggestion
for (let transformer of ['postcss', 'lightningcss']) {
```

Or, if it gives us any benefits, we could use `describe.each` as well: https://vitest.dev/api/#describe-each